contrib/devtools/lint-includes.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check for duplicate includes.
+
+EXIT_CODE=0
+for HEADER_FILE in $(git ls-files | grep -E '^src/.*\.h$' | grep -Ev '/(leveldb|secp256k1|univalue)/'); do
+    DUPLICATE_INCLUDES_IN_HEADER_FILE=$(grep -E '^#include ' < "${HEADER_FILE}" | sort | uniq -d)
```

Comment:
Why are you using redirection here?

Reply:
Do you mean why I'm storing the output in `DUPLICATE_INCLUDES_IN_HEADER_FILE`? :-)  That is the only redirection taking place here, right? :-)

I need the output to print it below. I'm guessing that's not what you meant? :-)